### DISCUSSION TITLE
how to write mol to mol2file??

### DISCUSSION THREAD

**Discussion by cgh2797** [rdkit/rdkit#3647] [Q&A] [answered]
I want to write mol of rdkit as mol2 file.
There is a MolToMolFile function, but it is saved only in a 2d structure.

I want to save in mol2 format in 3d, what should I do?

**Comment by greglandrum** [comment, marked as answer]
The RDKit is not able to write Mol2 files due to the ambiguity of the format definition.
Doing a bit of web searching will turn up numerous discussion about why this is the case.

**Reply by dalong12** [reply]
http://www.ccl.net/chemistry/resources/messages/2008/08/11.006-dir/
https://sourceforge.net/p/rdkit/mailman/rdkit-discuss/thread/e304012e-9683-bb25-5c07-e399b7edd18e%40unito.it/#msg35461346
